code/src/components/Loader.js

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+import styled from "styled-components"
+
+const LoaderStyled = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: 0;
+  bottom: 0;
+  }
+`
+
+const Loader = () => {
```

Comment:
Nice with a loader, but we can't see it on the page. Maybe the gif is loading slower?